[mingw64] dav1d build fails.

It suddenly fails yesterday.

error log:https://0x0.st/zpUh.zip


> [313m14:33:44(B[m [1m┌ dav1d git (B[m................................... [[311mFiles missing(B[m]
> [313m14:33:44(B[m [1m├(B[m Running meson...
> [313m14:33:51(B[m [1m├(B[m Running build...
> Likely error:
> C:/media-autobuild_suite/local64/lib/libshaderc_combined.a(validate_memory_semantics.cpp.o):validate_memory_semantics.cpp:(.text+0xb24): undefined reference to `std::basic_ios<char, std::char_traits<char> >::clear(std::_Ios_Iostate)'
> 
> C:/media-autobuild_suite/local64/lib/libshaderc_combined.a(validate_memory_semantics.cpp.o):validate_memory_semantics.cpp:(.text+0xb40): undefined reference to `std::basic_ios<char, std::char_traits<char> >::clear(std::_Ios_Iostate)'
> 
> C:/media-autobuild_suite/local64/lib/libshaderc_combined.a(validate_memory_semantics.cpp.o):validate_memory_semantics.cpp:(.text+0xb5c): undefined reference to `std::basic_ios<char, std::char_traits<char> >::clear(std::_Ios_Iostate)'
> 
> C:/media-autobuild_suite/local64/lib/libshaderc_combined.a(validate_memory_semantics.cpp.o):validate_memory_semantics.cpp:(.text+0xb78): undefined reference to `std::basic_ios<char, std::char_traits<char> >::clear(std::_Ios_Iostate)'
> 
> C:/media-autobuild_suite/local64/lib/libshaderc_combined.a(validate_memory_semantics.cpp.o):validate_memory_semantics.cpp:(.text+0xb94): undefined reference to `std::basic_ios<char, std::char_traits<char> >::clear(std::_Ios_Iostate)'
> 
> C:/media-autobuild_suite/local64/lib/libshaderc_combined.a(validate_memory_semantics.cpp.o):validate_memory_semantics.cpp:(.text+0xbe1): undefined reference to `std::basic_ostream<char, std::char_traits<char> >& std::operator<< <std::char_traits<char> >(std::basic_ostream<char, std::char_traits<char> >&, char const*)'
> 
> C:/media-autobuild_suite/local64/lib/libshaderc_combined.a(validate_memory_semantics.cpp.o):validate_memory_semantics.cpp:(.text+0xc5d): undefined reference to `std::__throw_out_of_range_fmt(char const*, ...)'
> 
> C:/media-autobuild_suite/local64/lib/libshaderc_combined.a(validate_memory_semantics.cpp.o):validate_memory_semantics.cpp:(.text+0xe): undefined reference to `std::basic_ios<char, std::char_traits<char> >::clear(std::_Ios_Iostate)'
> 
> collect2.exe: error: ld returned 1 exit status
> ninja: build stopped: subcommand failed.
> 
> [31mbuild failed. Check C:/media-autobuild_suite/build/dav1d-git/build-64bit/ab-suite.build.log(B[m
> [31mThis is required for other packages, so this script will exit.(B[m
> [313m14:34:40(B[m   Creating diagnostics file...
> 
> [32mAll relevant logs have been anonymously uploaded to https://0x0.st/zpUh.zip
> [32mCopy and paste [31m[logs.zip](https://0x0.st/zpUh.zip)[32m in the GitHub issue.(B[m
> Make sure the suite is up-to-date before reporting an issue. It might've been fixed already.
> Try running the build again at a later time.

--- a/build/media-suite_compile.sh
+++ b/build/media-suite_compile.sh
@@ -928,6 +928,18 @@ if { [[ $aom = y ]] || { [[ $ffmpeg != "no" ]] && enabled libaom; }; } &&
 fi
 unset _aom_bins
 
+_check=(dav1d/dav1d.h dav1d.pc libdav1d.a)
+[[ $standalone = y ]] && _check+=(bin-video/dav1d.exe)
+if { [[ $dav1d = y ]] || { [[ $ffmpeg != "no" ]] && enabled libdav1d; }; } &&
+    do_vcs "https://code.videolan.org/videolan/dav1d.git"; then
+    do_uninstall include/dav1d "${_check[@]}"
+    extracommands=()
+    sed -i 's/sdl2_dependency.found()/false/' tools/meson.build
+    [[ $standalone = y ]] || extracommands=(-Denable_tools=false)
+    do_mesoninstall video -Denable_tests=false "${extracommands[@]}"
+    do_checkIfExist
+fi
+
 _check=()
 { [[ $rav1e = y ]] ||
     { enabled librav1e && [[ $standalone = y ]]; }; } &&
@@ -1600,17 +1612,6 @@ if [[ $ffmpeg != "no" ]] && enabled liblensfun &&
         -DINSTALL_HELPER_SCRIPTS=off -DCMAKE_INSTALL_DATAROOTDIR="$LOCALDESTDIR/bin-video"
     do_checkIfExist
     add_to_remove
-fi
-
-_check=(dav1d/dav1d.h dav1d.pc libdav1d.a)
-[[ $standalone = y ]] && _check+=(bin-video/dav1d.exe)
-if { [[ $dav1d = y ]] || { [[ $ffmpeg != "no" ]] && enabled libdav1d; }; } &&
-    do_vcs "https://code.videolan.org/videolan/dav1d.git"; then
-    do_uninstall include/dav1d "${_check[@]}"
-    extracommands=()
-    [[ $standalone = y ]] || extracommands=(-Denable_tools=false)
-    do_mesoninstall video -Denable_tests=false "${extracommands[@]}"
-    do_checkIfExist
 fi
 
 _check=(bin-video/vvc/{Encoder,Decoder}App.exe)

